app/forms/Validation.scala

```diff
@@ -22,6 +22,7 @@ object Validation {
 
   val accountNumberPattern: Regex   = "^[ -]*(?:\\d[ -]*){6,8}$".r.anchored
   val sortCodePattern: Regex        = "^[ -]*(?:\\d[ -]*){6,6}$".r.anchored
+  val eoriPattern: Regex = "(?:[Gg][Bb])?\\d{12}".r.anchored
```

Comment:
The regex example we have inthe api doc is 
"([a-zA-Z]{2}[0-9]{15})|([a-zA-Z]{2}[0-9]{12})|([a-zA-Z]{4})" I think we should restrict the characters to 'GB/gb' as you've done but the length is different. Can we discuss?

Reply:
@lmugnai as advised I have done other changes, but not sure how to work with regex from API, will talk tomorrow.